pygpcca/_gpcca.py

```diff
@@ -973,10 +977,20 @@ def optimize(
                 )
             else:
```

Comment:
I think this is less-indented and more readable:
```python3
if len(self._p_eigenvalues) < m:
    raise ValueError(
        f"Can't check complex conjugate block splitting for {m} clusters with only "
        f"{len(self._p_eigenvalues)} eigenvalues."
    )
if _check_conj_split(self._p_eigenvalues[:m]):
    if len(m_list) == 1:
        raise ValueError(
            f"Clustering into {m} clusters will split complex conjugate eigenvalues. "
            f"Request one cluster more or less."
        )
    warnings.warn(
        f"Clustering into {m} clusters will split complex conjugate eigenvalues. "
        f"Skipping clustering into {m} clusters."
    )
    crispness_list.append(0)
    chi_list.append(np.zeros((n, m)))
    rot_matrix_list.append(np.zeros((m, m)))
    continue
```

Reply:
Also, it would be great to have a test for this.

Reply:
Agreed - I improved the readability by [911c92b](https://github.com/msmdev/pyGPCCA/pull/13/commits/911c92b5dcf028636c9ea42cd8729bf6342998de).
Additionally I fixed some things causing troubles when building the docs and improved some grammar in utils/_docs.py.

Regarding test: I added something in the example notebook utilizing the new behavior and properties - mb. you could take a look there and set up a small test for this. Currently, I am running out of time I can invest in the project (at least for today)...  -.-